aries-site/src/examples/templates/forms/SimpleSignUpExample.js

```diff
@@ -179,9 +214,7 @@ export const SimpleSignUpExample = () => {
                             <FormCheckmark size="small" />
                           </Box>
                         ) : (
-                          <Box alignSelf="center">
-                            <FormClose size="small" />
-                          </Box>
+                          <></>
```

Comment:
Same as other comment regarding swapping `?` `:` to `&&` because we don't need empty tag